packages/cozy-procedures/src/components/documents/menuUpload/MenuUploadWeb.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  translate,
+  Menu,
+  MenuItem,
+  Button,
+  Icon,
+  Caption,
+  FileInput
+} from 'cozy-ui/transpiled/react'
+
+import UploadInputLabel from '../UploadInputLabel'
+
+class MenuUploadWeb extends Component {
+  render() {
+    const { t, onChange } = this.props
+    return (
+      <Menu
+        component={
+          <Button theme="ghost" extension="full" align="left">
+            <Icon icon="plus" size={16} className="u-mr-1 u-pa-half" />
+            <span>{t('documents.import')}</span>
+          </Button>
+        }
+        className="u-db"
+        position={'right'}
+        /**
+         * We need to stop the propagation since when we click on an Menu, since auto-close
```

Comment:
remove this comment (or maybe fix it?)

Reply:
In fact, we stop the propagation by doing a onSelect=false. Will update the commit to reflect it better 👍